nomos-services/cryptarchia-consensus/src/lib.rs

```diff
@@ -421,12 +433,17 @@ where
         )
         .await;
 
+        let time_relay: OutboundRelay<_> = self
```

Comment:
It'd be nice if you could add this to the `CryptarchiaConsensusRelays` struct.

Speaking of which, does it make sense to also move the block broadcaster into it?

Reply:
In reality this relay is consumed and replaced by the `slot_stream` and not used by anything else. So probably not work to keep a reference to it to be sent to other methods if there is no need. We can add it later on if we add more functionality to it.